internal/cmd/about.go

```diff
@@ -14,7 +14,6 @@ type triangle struct {
 	B     *point3D
 	C     *point3D
 	Color rune
-	surf  *sprite.Surface
```

Comment:
This was failing linting for me locally. I'm not sure why it doesn't fail the linter in CI, but it appears to be unused.

Reply:
Weird. It didn't pick up w/ lint for me either. It's safe to remove.